apphelpers/rest/hug.py

```diff
@@ -27,6 +30,21 @@ def wrapper(*ar, **kw):
     return f
 
 
+def honeybadger_wrapper(f, hb):
```

Comment:
@mandarvaze  here we can refer `dbtransaction` wrapper
`def honeybadger_monitoring(hb):`